version.props

```diff
@@ -6,6 +6,7 @@
     <PreReleaseLabel>preview</PreReleaseLabel>
     <PreReleaseBrandingLabel>Preview</PreReleaseBrandingLabel>
     <ExperimentalVersionPrefix>0.3.$(AspNetCorePatchVersion)</ExperimentalVersionPrefix>
+    <BlazorComponentsVersionPrefix>0.8.$(AspNetCorePatchVersion)</BlazorComponentsVersionPrefix>
```

Comment:
@SteveSandersonMS should blazor packages remain 0.8.0?

Reply:
As far as I know, yes. We'll switch to a 1.0.0-x (or 3.x) numbering scheme once we're approved to do so, but that's not yet.